packages/razorsharp/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "figma-plugin-blade",
+  "version": "1.0.0",
+  "description": "",
+  "main": "code.js",
+  "scripts": {
+    "build": "NODE_ENV=production webpack",
+    "dev": "NODE_ENV=development webpack --watch",
+    "tsc": "tsc --noEmit",
+    "test": "jest",
+    "test:watch": "jest --watch"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "@figma/plugin-typings": "^1.58.0",
+    "@types/jest": "^29.5.0",
+    "css-loader": "^6.7.3",
+    "html-webpack-plugin": "^5.5.0",
+    "identity-obj-proxy": "^3.0.0",
+    "inline-chunk-html-plugin": "^1.1.1",
+    "jest": "^29.5.0",
+    "style-loader": "^3.3.1",
+    "ts-jest": "^29.0.5",
+    "ts-loader": "^9.4.2",
+    "ts-node": "^10.9.1",
+    "typescript": "^4.9.4",
+    "webpack": "^5.76.0",
+    "webpack-cli": "^5.0.1"
+  },
+  "dependencies": {
+    "clipboard-copy": "^4.0.1",
+    "preact": "^10.13.0"
```

Comment:
No react?

Reply:
Plugin uses Preact only for UI. React + ReactDOM would add another 45kb to the plugin bundle. File size is important for the plugin so there has to be a strong reason for using React over Preact